ports/juce/vcpkg.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "juce",
+  "version": "7.0.7",
+  "description": "Open-source cross-platform C++ application framework for desktop and mobile applications.",
+  "homepage": "https://juce.com",
+  "license": null,
+  "supports": "!android & !uwp",
+  "dependencies": [
```

```suggestion
  "dependencies": [
    {
      "name": "juce",
      "host": true
    },
```

`juce` should depend upon itself to build the host tools.